intro-to-R-tidyverse/01b-intro_to_base_R_exercise.Rmd

```diff
@@ -18,7 +18,7 @@ We'll start with some basic manipulations of values and variables.
 In the chunk below calculate the product of 1.563 and 4.678.
 
 ```{r multiply, solution = TRUE}
-
+1.563 * 4.678
```

Comment:
? Is this supposed to be here?

Reply:
No! This is an exercise notebook!

Reply:
Ah yes will remove! (Leftover from an example being shown during the PGH workshop)